### DISCUSSION TITLE
Zoom Meeting Series

### DISCUSSION THREAD

**Discussion by dthaler** [microsoft/ebpf-for-windows#427] [Announcements]
There is a weekly public meeting series for ebpf-for-windows every Monday at 8:30am Pacific. This meeting will go over any new github issues not yet discussed that people want to discuss, and the meeting will end when we run out of issues, so might only be 10 minutes long, or might take up to an hour.

Topic: ebpf-for-windows issue triage
Time: Every week on Mon 8:30 AM Pacific time
Please download and import the following iCalendar (.ics) files to your calendar system.
Weekly: https://zoom.us/meeting/tJYtdeqorjssGdDHI8iVeb6qgIfuWH4iPQ9y/ics?icsToken=98tyKuCqrjkpGtSVthqDRowQAojob_TziGJdjad5tDa9ChZWYFXnMctEJ4VXRNHg

Join Zoom Meeting
https://zoom.us/j/92023000634?pwd=a2NOUFN5YkVPcG9aaVNGYVlDd2dmUT09

Meeting ID: 920 2300 0634
Passcode: 448433
One tap mobile
+12532158782,,92023000634#,,,,*448433# US (Tacoma)
+13462487799,,92023000634#,,,,*448433# US (Houston)

Dial by your location
+1 253 215 8782 US (Tacoma)
+1 346 248 7799 US (Houston)
+1 669 900 6833 US (San Jose)
+1 312 626 6799 US (Chicago)
+1 929 436 2866 US (New York)
+1 301 715 8592 US (Washington DC)
Meeting ID: 920 2300 0634
Passcode: 448433
Find your local number: https://zoom.us/u/aqRGyAuua

**Comment by dthaler** [comment]
When there is a US holiday on a Monday, this meeting will be on Tuesday morning.  For example, since Sept. 6, 2021 is Labor Day, the triage meeting will be on Tuesday September 7, same time and Zoom link.

**Reply by dthaler** [reply]
September 20th's meeting is canceled as it is during the Linux Plumbers Conference, so as not to conflict with other eBPF events.

**Reply by dahavey** [reply]
Schedule: Dec. 20th, 27th and Jan 3 cancelled for holidays. Reconvene on Jan 10th.

**Reply by dahavey** [reply]
Just a reminder that this morning's meeting is canceled. Happy holidays!

**Comment by dahavey** [comment]
One more holiday reminder. Regular meetings will resume on Jan 10th.

**Reply by dthaler** [reply]
Jan 17th is a US holiday (MLK Day) and there's only 2 new items since last week so there will be no meeting on the 17th.  Enjoy the holiday.

**Reply by dthaler** [reply]
February 21st is another US holiday (Presidents Day).  At the end of today's meeting we decided to skip next week so the next meeting will be February 28th.  Enjoy the holiday.

**Comment by rob-scheepens** [comment]
Just fyi: when I downloaded the ics on my Mac and opened it, this is what Outlook gave me:

<img width="488" alt="image" src="https://user-images.githubusercontent.com/15910576/150325531-48fde28f-64f6-4e5d-aa6e-8088d0a0e9db.png">

The meeting is not added to my calendar, so I added it manually.

**Comment by dthaler** [comment]
Daylight savings time just started in the US, so our triage meetings just shifted by an hour for those elsewhere.  The meetings are still 8:30 AM Pacific time at the usual link.

**Comment by dthaler** [comment]
Monday May 30, 2022 is a US Holiday (Memorial Day) so there will be no ebpf-for-windows meeting that day.   We will resume as usual on Monday June 6

**Comment by dthaler** [comment]
July 4th is a US Holiday so there will be no ebpf-for-windows meeting that day,

**Comment by dahavey** [comment]
Due to folks being out for the IETF we will cancel the triage meeting on Monday 7/25. We will resume meeting the following Monday 08/01.

**Comment by dahavey** [comment]
Due to Labor day (US Holiday) September 5, 2022 triaged is skipped for the week.

**Comment by dthaler** [comment]
Due to Cloud Native eBPF Day, October 24, 2022 triage meeting is skipped for next week.  We encourage all to attend [Cloud Native eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).  Thanks to the eBPF Foundation for providing a virtual option for those who cannot attend in person!

**Comment by dahavey** [comment]
Good moment everyone!
Monday Feb 20th is President's Day a U.S. Holiday. Triage is cancelled and we will reconvene the following Monday on Feb 27th at 8:30 am Pacific Time. Looking forward to seeing everyone there.

**Comment by hawkinsw** [comment]
Sorry to bother -- I just didn't know how Microsoft treated Juneteenth!

**Comment by dahavey** [comment]
Tuesday July 4th is a US holiday and many folks are taking Monday OOF. As a result we will be cancelling triage on Monday July 3rd and picking up again the following week.
Thanks, and happy holiday!

**Comment by dahavey** [comment]
We are planning on opening up office hours for the public. We are looking at 8am-8:30am Pacific time or 9:30am-10am Pacific time on Monday. (Either before or after the triage meeting)

Please indicate your preference for timing in this chat.

**Reply by dthaler** [reply]
To clarify, the Zoom meeting series here has been public office hours and is listed as such at https://ebpf.io/infrastructure/
However, we often spend more than half the time, and occasionally all the time, on issue triage rather than open discussion.
So Daniel is asking that if we extend the time to make sure we have time for any open discussion, should we start earlier or end later?

**Comment by shankarseal** [comment]
Monday September 4th, Labor Day is a U.S. Holiday. Triage is cancelled and we will reconvene the following Monday on September 11th at 8:30 am Pacific Time.

**Comment by dahavey** [comment]
Because of the Holidays our next triage meeting will be on January 8th.

**Reply by dthaler** [reply]
We have 2 people on the triage meeting channel now but don't see any Microsoft people on.  It's a US holiday but there's no cancellation notice here or in slack.

**Reply by dthaler** [reply]
In 2021 above, it was stated that the policy is "When there is a US holiday on a Monday, this meeting will be on Tuesday morning."  Is that still the case?

**Reply by dahavey** [reply]
Yes, that is the case.

**Comment by dahavey** [comment]
February 19th is a US Holiday so the Monday eBPF for Windows triage meeting will be held on Tuesday February 20th at the same time as per policy.

**Comment by dahavey** [comment]
May 27th is a US Holiday so the Monday eBPF for Windows triage meeting will be held on Tuesday May 28th at the same time as per policy.

**Comment by shankarseal** [comment]
September 2, 2024 is a US holiday (Labor Day). So the monday eBPF for Windows triage meeting will be held on September 3rd, at 8:30 AM PDT.

**Comment by shankarseal** [comment]
There will not be any eBPF Issues Triage meetings on December 23rd and December 30th, 2024.
Happy Holidays to all!

**Comment by shankarseal** [comment]
There will not be any Issues Triage meetings/Office hours on January 20, 2025, due to US Holiday (Martin Luther King Day).

**Comment by shankarseal** [comment]
There will not be any Issues Triage meetings/Office hours on February 17, 2025, due to US Holiday (President's Day).

**Comment by shankarseal** [comment]
Monday 5/26 is a US holiday (Memorial day). Hence the eBPF triage meeting scheduled for that day will instead be held on Tuesday 5/27 at 8:30 AM PDT.

**Comment by shankarseal** [comment]
September 1, 2025 is a US holiday (Labor Day). So, the eBPF for Windows triage meeting will be held on September 2nd, at 8:30 AM PDT.

**Comment by shankarseal** [comment]
January 19, 2026 is a US holiday (Martin Luther King day). So, the eBPF for Windows triage meeting will be held on January 21st, at 8:30 AM PST.

**Comment by shankarseal** [comment]
Februrary 16, 2026 is a US holiday (President's Day). So, the eBPF for Windows triage meeting will be held on Februrary 17th, at 8:30 AM PST.

**Comment by shankarseal** [comment]
Monday 25th of May, 2026 is a US Holiday (Memorial Day). So, the eBPF for Windows Triage meeting will be cancelled for that day.

**Comment by shankarseal** [comment]
September 7, 2026 is a US holiday (Labor Day). So, the eBPF for Windows triage meeting will be instead held on Tuesday, September 8th, at 8:30 AM PDT.
